### PR TITLE
Porting to 8.15

### PR DESCRIPTION
**Pull request by MevenBertrand**
Mostly to get a CI run.

### CODE REVIEW

**Comment by MevenBertrand** [comment]
Nevermind, there's no CI…

File: monadic/library/zipSeq.ml [hunk 1]
```diff
@@ -1,13 +1,13 @@
 module ZipSeqAlternative = struct
   type 'a t = 'a Stdlib.Seq.t
 
-  let zip xs ys () =
+  let rec zip xs ys () =
```
Comment (kyoDralliam) [inline monadic/library/zipSeq.ml:4]: I think this `rec` is ambiguous: for `Stdlib` version `< 4.14`, `Stdlib.Seq.zip` is not defined  hence the recursive call is used, but in versions `>= 4.14` the function `zip` is defined in the lib and this `rec` is unused... And similar for `drop` below.

Comment (MevenBertrand) [reply monadic/library/zipSeq.ml:4]: Yeah, it's been a source of confusion and back and forth issues… I am no OCaml expert, do you know how to disambiguate to give priority to the local recursively defined function? Or maybe we should drop (this part of) the monadic library, which seems to be stuck with 4.13?

**Comment by yforster** [comment]
done, I pushed your branch to this repo now

**Pull request closed without merging**